lib/shelley/src/Cardano/Wallet/Shelley/Network/Blockfrost.hs

```diff
@@ -965,6 +965,7 @@ fromBlockfrostPP network pp@BF.ProtocolParams{..} = do
                     , protocolParamCollateralPercent = Just collateralPercent
                     , protocolParamMaxCollateralInputs =
                         Just $ intCast maxCollateralInputs
+                    , protocolParamUTxOCostPerByte = Just 4310
```

Comment:
Suggestion: locate this protocol parameter value next to `protocolParamUTxOCostPerWord`.

Reply:
Similarly, it's possible to import this value from here:
https://github.com/input-output-hk/cardano-wallet/blob/1f72c95d081c973e492da739faeee4bdefcfd244/lib/core/src/Cardano/Wallet/Primitive/Types/MinimumUTxO/Gen.hs#L162